modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/application/authz/ApplicationAuthzTestCase.java

```diff
@@ -83,6 +86,9 @@
                     "    <Rule Effect=\"Deny\" RuleId=\"denyall\"/>\n" +
                     "</Policy>";
 
+    CookieStore cookieStore = new BasicCookieStore();
+    CookieStore cookieStoreUnAuthorized = new BasicCookieStore();
```

Comment:
do we need to declare these variables? cant we pass like  
`httpClientAzUser = HttpClientBuilder.create().setDefaultCookieStore(new BasicCookieStore()).build();`

Reply:
in other places as well..

Reply:
In some places, we can reuse the cookie store by clearing it like:  `cookieStore.clear()`, I will fix the other places.